01_fib/willshen8/fib_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
Add tests for `n == 0`  and `n < 0` - do something reasonable (e.g. print nothing), but not print the wrong output.

Reply:
Thank you, this has been handled.